lorentz_force/lorentz_unit_tests.py

```diff
@@ -0,0 +1,49 @@
+"""
+Module to test lorentz_force.py
+"""
+
+import math
+from lorentz_force import lorentz_force
+
+def test_lorentz_force_at_90_degrees():
+    """
+    Test case where the angle between velocity and magnetic field is 90 degrees (π/2 radians).
+    The sin(90 degrees) is 1, so the force should be maximized.
+    """
+    charge = 1.6e-19  # Charge in Coulombs
+    velocity = 2e6    # Velocity in m/s
+    magnetic_field = 0.01  # Magnetic field in Tesla
+    angle_radians = math.pi / 2  # 90 degrees
+
+    expected_force = charge * velocity * magnetic_field  # sin(90 degrees) is 1
+    assert math.isclose(lorentz_force(
```

Comment:
Missing tolerance in math.isclose()

Reply:
What should the tolerance be? By default it uses a tolerance of 1e-9

Reply:
upon more research the default tolerance of 1e-9 is generally sufficient for many computational physics problems